src/Elmish.WPF/DynamicViewModel.fs

```diff
@@ -55,31 +55,26 @@ type [<AllowNullLiteral>] internal DynamicViewModel<'model, 'msg>
     log.LogTrace("[{BindingNameChain}] ErrorsChanged {BindingName}", nameChain, name)
     errorsChanged.Trigger([| box this; box <| DataErrorsChangedEventArgs name |])
 
+  let getFunctionsForSubModelSelectedItem selectedItemTargetBindings (name: string) =
```

Comment:
I don't like the name `selectedItemTargetBindings` because it isn't accurate.  Accurate and precise would be `initializedBindings` or less precisely just `bindings`.

Reply:
I like initializedBindings because it will hold up nicely in the final version.

Reply:
Updated to `initializedBindings`